src/Phoenix/PhoenixClient.ts

```diff
@@ -86,13 +96,26 @@ export class PhoenixClient {
             status: ContainerStatusType.Active,
           };
         } else if (response.status === HttpStatusCodes.NotFound) {
+          TelemetryProcessor.traceMark(Action.PhoenixReconnect, {
+            dataExplorerArea: Areas.Notebook,
```

Comment:
>dataExplorerArea [](http://example.com/codeflow?start=12&length=16)

would this have db account and puid so we can get distinct count of user/accounts having running into this issue

Reply:
Also - as informed, we  need log for when reconnect happens.

Reply:
> > dataExplorerArea
> 
> would this have db account and puid so we can get distinct count of user/accounts having running into this issue

Yes the record will have the details